deep_qa/data/instances/sentence_selection_instance.py

```diff
@@ -134,20 +133,21 @@ def pad(self, max_lengths: Dict[str, int]):
         as well as the individual words in the questions and sentences themselves.
         """
         max_lengths_tmp = max_lengths.copy()
-        # pad the number of words in a question
-        max_lengths_tmp['num_sentence_words'] = max_lengths_tmp['num_question_words']
-        self.question_indices = self.pad_word_sequence(self.question_indices, max_lengths_tmp)
-        # pad the number of sentences
+        # Pad the number of sentences.
         num_sentences = max_lengths['num_sentences']
         while len(self.sentences_indices) < num_sentences:
             self.sentences_indices.append([])
         self.sentences_indices = self.sentences_indices[:num_sentences]
 
-        # pad the number of words in a sentence
-        max_lengths_tmp['num_sentence_words'] = max_lengths_tmp['num_passage_words']
+        # Pad the number of words in a sentence.
+        # Note that max_lengths_tmp['num_sentence_words'] is set to
+        # ['num_sentence_words'] by default.
```

Comment:
I get that you're trying to say that you're using the key defined by `TextTrainer` here, but I don't think the comment makes much sense as-is.